src/components/MapboxMap/MapboxMap.vue

```diff
@@ -85,6 +92,29 @@
       onMapLoaded () {
         this.$store.commit('ui/SET_MAP_READY', true)
       },
+
+      async externalGeocoder (query) {
+        const reservoirs = await this.$repo.reservoir.getAll({
+          params: {
+            limit: 1,
+            ids: [query],
+          },
+        }).catch(() => null)
```

Comment:
Ignoring the error here, since it also returns an error when a searched ID does not exist